Configuration/src/ConfigurationProviders/README.md

```diff
@@ -1,16 +1,16 @@
-﻿# Steeltoe Configuration Providers Sample App
+# Steeltoe Configuration Providers Sample App
 
 This is an ASP.NET Core application that shows how to use various `IConfiguration` providers that are part of the Steeltoe project.
 
 ## General pre-requisites
 
 1. Installed .NET 8 SDK
-1. Optional: [VMware Tanzu Platform for Cloud Foundry](https://docs.vmware.com/en/VMware-Tanzu-Application-Service/index.html)
-   (optionally with [Windows support](https://docs.vmware.com/en/VMware-Tanzu-Application-Service/6.0/tas-for-vms/concepts-overview.html))
-   with [Spring Cloud Services for VMware Tanzu](https://docs.vmware.com/en/Spring-Cloud-Services-for-VMware-Tanzu/index.html)
-   and [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
-1. Optional: [VMware Tanzu Platform for Kubernetes](https://docs.vmware.com/en/VMware-Tanzu-Platform/services/create-manage-apps-tanzu-platform-k8s/overview.html) v1.5 or higher
-   with [Application Configuration Service](https://docs.vmware.com/en/Application-Configuration-Service-for-VMware-Tanzu/index.html)
+1. Optional: [Tanzu Platform for Cloud Foundry](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/concepts-overview.html)
+   (optionally with [Windows support](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/toc-tasw-install-index.html))
+   with [Spring Cloud Services for Cloud Foundry](https://techdocs.broadcom.com/us/en/vmware-tanzu/spring/spring-cloud-services-for-cloud-foundry/3-3/scs-tanzu/index.html)
+   and [Cloud Foundry CLI](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/6-0/tpcf/cf-cli-index.html)
+1. Optional: [Tanzu Platform for Kubernetes](https://techdocs.broadcom.com/us/en/vmware-tanzu/standalone-components/tanzu-application-platform/1-12/tap/overview.html) v1.5 or higher
+   with [Application Configuration Service](https://techdocs.broadcom.com/us/en/vmware-tanzu/standalone-components/application-configuration-service-for-tanzu/2-4/app-config-service/overview.html)
    and [Kubernetes](https://kubernetes.io/docs/tasks/tools/)
```

Comment:
I think the "and Kubernetes" bit might be distracting or confusing. What do you think about removing it from this page and replacing it on the Connectors pages with "or another platform that supports the Service Binding Specification for Kubernetes"?

Reply:
Well, we provide specific instructions below, and in order to follow them, k8s needs to be installed, so it _is_ a prerequisite, right?

Reply:
It's been a while ago, but I recall I had to install k8s tooling (k9s and others) before I could use commands like `tanzu app workload ...`. From what I understood, k8s was a prerequisite. Is that not the case (anymore), or did I misunderstand something? Perhaps I had to install it just to see what `tanzu app` commands were doing?

Reply:
k9s definitely would have been optional. [The Kubernetes CLI is a pre-req](https://techdocs.broadcom.com/us/en/vmware-tanzu/standalone-components/tanzu-application-platform/1-12/tap/prerequisites.html#tools-and-cli-reqs) for TAP, and it looks like we're using the same link here, but with our link specifying it as "Kubernetes" it's unclear what of the 4 tools on that page should be installed... If we change it to "the Kubernetes CLI" or "kubectl" then it would make more sense to keep